src/components/skills-quiz/CardLoadingSkeleton.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Card } from '@edx/paragon';
+import Skeleton from 'react-loading-skeleton';
+
+const CardLoadingSkeleton = () => (
+  <div className="row col col-12 p-0">
+    <div className="skill-quiz-results align-items-l-between col col-xl-10">
+      {Array.from({ length: 9 }, (_, i) => (
```

Comment:
Can you please explain how will this work. Are we showing 9 skeleton course cards while loading results? Also, I think we should use a constant here instead of hardcoding this value